x/data/client/testsuite/cli_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
```

Comment:
This file should be renamed to e.g. `x/data/client/testsuite/suite.go` (no _test suffix)